.editorconfig

```diff
@@ -2,12 +2,15 @@ root = true
 
 [*]
 indent_size = 2
+indent_style = space
```

Comment:
this isn't in the gradle config below.  revert?  or am i missing something?

Reply:
I added it to solidify the setting, my IDEA keeps inserting tabs without this, every time I paste code, indent, etc. because my personal code style uses that and the project does not specify it. I would imagine this applies to any contributor using tabs.

Reply:
no.